src/components/DBSearchResults.vue

```diff
@@ -207,6 +210,7 @@
             <span v-if="rating.medium && rating.date">on</span>
             <span v-else-if="rating.date">On</span>
             {{formattedDate(rating.date)}}
+            <span style="font-size: 12px; color: rgb(167, 167, 167);" v-if="rating.tags">&nbsp;&nbsp;{{ turnArrayIntoList(rating.tags, "title") }}</span>
```

Comment:
Instead of this inline styling you should put these styles into the style tag at the bottom of this file. If you look at how those styles are arranged, you'll see that the nesting structure of the classes in the style tag match the nesting of the html elements in the template. Following this pattern, you should move these styles down there.

In particular, I would give this element a new class (maybe `class="ratings-tags"`) and then move your styles into a new rule nested appropriately (inside of `.full-info`, maybe a sibling of `.rating-categories` and `.actors`). 

I'm surprised that the font-size needs changing, it seems like it should match the other text in this spot but if it looks good like this, that's fine. You should use `rem` as the unit for font and font based things because `rem`s scale for accessibility devices. 1rem = 16px by default so 12px is `0.75rem`.

The color should be in hex values to be consistent `rgb(167, 167, 167)` is `#A7A7A7`.

Finally, the two `nbsp;`s can probably be achieved with a `padding-left` rule which would be better.